pkg/cli/tsdump_upload.go

```diff
@@ -470,14 +477,16 @@ func (d *datadogWriter) upload(fileName string) error {
 	}
 	fmt.Printf("\nUpload status: %s!\n", uploadStatus)
 	fmt.Printf("Uploaded %d series overall\n", seriesUploaded)
+	fmt.Printf("Uploaded %d unique series overall\n", len(allMetrics))
+
 	// Estimate cost. The cost of historical metrics ingest is based on how many
 	// metrics were active during the upload window. Assuming the entire upload
 	// happens during a given hour, that means the cost will be equal to the count
 	// of uploaded series times $4.55/100 custom metrics (our rate) divided by
 	// 730 hours per month.
 	// For a single node upload that has 6500 unique series, that's about $.40
 	// per upload.
-	estimatedCost := float64(seriesUploaded) * 4.55 / 100 / 730
+	estimatedCost := float64(len(allMetrics)) * 4.55 / 100 / 730
```

Comment:
Ohh this seems to reduce the estimate significantly :P

---

Does the number of datapoints not matter at all? only the number of series?